unit/src/InputParametersTest.C

```diff
@@ -301,3 +301,68 @@ TEST(InputParameters, setPPandVofPP)
 
   // EXPECT_EQ(p1.getDefaultPostprocessorValue("pp_name"), 1);
 }
+
+TEST(InputParameters, getPairs)
```

Comment:
love the unit testing

Reply:
actually, I should add a unit test for the MultiMooseEnum case...

Reply:
done